src/util/common-mixin.js

```diff
@@ -266,17 +266,30 @@ export default {
         (x) => this.sdkExtClusterCode(x) === clusterRoleName
       )
     },
+    createLogoSrc(isTiny, category, isSelected = false) {
+      return (
+        (isTiny ? '/logo/tiny/' : '/logo/') +
+        category +
```

Comment:
What happens when category is empty string (''). Some of our very old GSDKs did not have category.